1.6.2 websockets access logs broken

```
web-1  | [ERROR] Application callable raised an exception
web-1  | Traceback (most recent call last):
web-1  |   File "/usr/local/lib/python3.12/site-packages/granian/_futures.py", line 4, in future_watcher
web-1  |     await inner(watcher.scope, watcher.proto)
web-1  |           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
web-1  |   File "/usr/local/lib/python3.12/site-packages/granian/asgi.py", line 130, in _logger
web-1  |     return _ws_logger(scope, proto)
web-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^
web-1  |   File "/usr/local/lib/python3.12/site-packages/granian/asgi.py", line 124, in _ws_logger
web-1  |     access_log(time.time(), scope, 101)
web-1  |   File "/usr/local/lib/python3.12/site-packages/granian/asgi.py", line 150, in access_log
web-1  |     'method': scope['method'],
web-1  |               ~~~~~^^^^^^^^^^
web-1  | KeyError: 'method'
```

--- a/granian/asgi.py
+++ b/granian/asgi.py
@@ -147,7 +147,7 @@ def _build_access_logger(fmt):
                 'protocol': 'HTTP/' + scope['http_version'],
                 'path': scope['path'],
                 'qs': scope['query_string'],
-                'method': scope['method'],
+                'method': scope.get('method', '-'),
                 'scheme': scope['scheme'],
             },
             resp_code,

